### ISSUE TITLE
Phpcs is not being supported anymore

### ISSUE THREAD

**Issue by leonardola** [jpagano/wordpress-vscode-extensionpack#2]
the PHPcs included in this pack is not being maintained anymore but there are alternatives like this one https://github.com/shevaua/vscode-phpcs

**Comment by themorgantown** [comment]
This is true, can we remove it? Would you prefer a pull request @jpagano ?

**Comment by jpagano** [comment]
Go for it @themorgantown 
Is this alternative still recommended?
